examples/chat-memory/src/tests.tsx

```diff
@@ -0,0 +1,165 @@
+import path from "path";
+import { Readable } from "stream";
+
+import * as gensx from "@gensx/core";
+import { OpenAIProvider } from "@gensx/openai";
+import { BlobProvider, useBlob } from "@gensx/storage";
+
+// Define our own chat message type structure that is compatible with OpenAI's API
+interface ChatMessage {
+  role: "system" | "user" | "assistant";
+  content: string;
+}
+
+// Main chat component that uses memory
+const TestMemory = gensx.Component<{ id: string }, string>(
```

Comment:
Should we either delete this file or turn this suite into vitest tests?

Reply:
fair, it's a bit messy. i'll remove it and just keep it for local testing. At some point we should probably set up some e2e tests.